### PR TITLE
update old-ethermint-docker-compose.yml to not use static genesis.json

### PR DESCRIPTION
**Pull request by jbowen93**
The ghcr.io/celestiaorg/ethermint:pre-v0.2.0 image doesn't have the changes from https://github.com/celestiaorg/ethermint/commit/f843b79de1f58dd1db806a54345c86f95865c6c8 so it cannot use a static genesis file.

### CODE REVIEW

File: docker/old-ethermint-docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ services:
       - HOME_DIR=/ethermint
     command: [
             "ethermintd", "start",
-            "--home", "/ethermint",
```
Comment (adlerjohn) [inline docker/old-ethermint-docker-compose.yml:11]: Is this change related?

Comment (jbowen93) [reply docker/old-ethermint-docker-compose.yml:11]: Yes because if we include the volume it is mounted at `/ethermint` whereas the default location it looks for a config is `/.ethermintd`. But this PR can be ignored in favor of https://github.com/celestiaorg/ephemeral-cluster/pull/22 which also includes this small change.
